src/web/app/components/question-types/question-edit-details-form/rank-recipients-question-edit-details-form.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="col-8 form-check padding-left-15px">
+  <div class="col-sm-8 col-12 form-check padding-left-15px">
```

Comment:
The class is `padding-left-15px`, but in the `.scss` file it's set to `padding-left: 50px`... I think either change the class to reflect the actual padding, or just make the switch directly to Bootstrap styles

Reply:
Good catch hahaha, will fix this! thanks